src/cockpit/tests/components/CEditTrip.test.ts

```diff
@@ -4,18 +4,37 @@
 import CEditTrip from "@/components/CEditTrip.vue";
 import { mockTripObj } from "@/tests/mockData/data";
 
-import { shallowMount } from "@vue/test-utils";
-import test from "ava";
+import { mount } from "@vue/test-utils";
+import { expect } from "@jest/globals";
 
 const mockedTrip = new mockTripObj();
 
-test("renders empty component", (t) => {
-  const trip = mockedTrip.trip;
-  const wrapper = shallowMount(CEditTrip, {
-    props: {
-      trip: trip,
-    },
-    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-  } as any);
-  t.regex(wrapper.find(".editLabel").text(), /City:/);
+describe("CEditTrip", () => {
+  it("renders component", () => {
+    const trip = mockedTrip.trip;
+    const wrapper = mount(CEditTrip, {
+      props: {
+        trip,
```

Comment:
`isNew` is a required param that caused the typecheck error.

```suggestion
        trip: trip,
        isNew: false,
```